src/conversions.jl

```diff
@@ -965,8 +965,8 @@ function to_rotation(s::VecTypes{N}) where N
     end
 end
 
-to_rotation(s::Tuple{VecTypes, AbstractFloat}) = qrotation(to_ndim(Vec3f, s[1], 0.0), s[2])
-to_rotation(angle::AbstractFloat) = qrotation(Vec3f(0, 0, 1), Float32(angle))
+to_rotation(s::Tuple{VecTypes, Number}) = qrotation(to_ndim(Vec3f, s[1], 0.0), s[2])
+to_rotation(angle::Number) = qrotation(Vec3f(0, 0, 1), angle)
```

Comment:
```suggestion
to_rotation(s::Tuple{VecTypes, Real}) = qrotation(to_ndim(Vec3f, s[1], 0.0), s[2])
to_rotation(angle::Real) = qrotation(Vec3f(0, 0, 1), angle)
```
We don't want people accidentally passing complex values to rotation, so better to head It off here.